ocean_provider/run.py

```diff
@@ -25,11 +38,23 @@ def get_version():
 
 @app.route("/")
 def version():
+    """
+    Contains the provider data for an user:
+        - software;
+        - version;
+        - network url;
+        - provider address;
+        - services endpoints, which has all
+        the existing endpoints from routes.py
+        which are not in blocked_url.
+    """
+
     info = dict()
     info['software'] = Metadata.TITLE
     info['version'] = get_version()
     info['network-url'] = config.network_url
     info['provider-address'] = get_provider_wallet().address
+    info['servicesEndpoints'] = get_services_endpoints()
```

Comment:
one request, can you please rename the key `servicesEndpoints` to `serviceEndpoints`